server/auth/session.guard.ts

```diff
@@ -3,9 +3,12 @@ import { Configuration, FrontendApi } from "@ory/client";
 import { Reflector } from "@nestjs/core";
 import { ConfigService } from "@nestjs/config";
 import { Roles } from "./ability/ability.factory";
+import { Logger } from "@nestjs/common";
 
 @Injectable()
 export class SessionGuard implements CanActivate {
+    private readonly logger = new Logger(SessionGuard.name);
```

Comment:
Question: what's the reason behind using Nestjs Logger in some classes and Winston logger and other classes ?